src/Map.js

```diff
@@ -987,7 +987,7 @@ export default class Map {
     log.log('current center:', center)
     const zoom_level = this.map.getZoom()
     const res = await this.requester.request({
-      url: `${this.apiServerUrl}nearest?zoom_level=${zoom_level}&lat=${center.lat}&lng=${center.lng}`,
+      url: `${this.apiServerUrl}nearest?zoom_level=${zoom_level}&lat=${center.lat}&lon=${center.lon}`,
```

Comment:
@RubenSmn seems this requirs the change on API side too?

Reply:
The problem is also that the `center` does return a `lng`

Reply:
Okay, we can do this way: revert this PR and re-create it, then let's go to the API side and finish the correction, then when can merge this PR, overwise, this PR will cause problem, right?

Reply:
Could we update the API so that this will be fixed?

Reply:
I created a PR for the API https://github.com/Greenstand/treetracker-web-map-api/pull/353

Reply:
Great, I'll take a look, but I vaguely remember it's the PostgreSQL return `lon` so we align with it's convention maybe, you can try this:

```
select region_id id, ST_ASGeoJson(centroid) centroid from active_tree_region, region 
where region.id = active_tree_region.region_id
```

To check the return json from postgresql

Reply:
Maybe there are some postgresql function can do some traslation

Reply:
@dadiorchen just checked and the test is passing!

Reply:
@dadiorchen @RubenSmn is there anything i have to do?